admin-ui/plugins/auth-server/components/Scopes/types/componentTypes.ts

```
@@ -0,0 +1,54 @@
+/**
+ * Props for ScopeDetailPage component
+ */
+export interface ScopeDetailPageProps {
+  row: Scope
+}
+
+/**
+ * Props for ScopeListPage component
+ * Currently no props needed (top-level page component)
+ */
+export interface ScopeListPageProps {}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)